scenes/enemies/guard/light.gd

```diff
@@ -0,0 +1,52 @@
+@tool
+extends PointLight2D
+
+@warning_ignore("unused_private_class_variable")
```

Comment:
Can you explain why this warning ignore is needed?

Reply:
Same as https://github.com/endlessm/threadbare/pull/47#discussion_r2016787760